cdap-examples/LogAnalysis/src/test/java/co/cask/cdap/examples/loganalysis/LogAnalysisAppTest.java

```diff
@@ -57,7 +50,7 @@
   private static final String TOTAL_HITS_VALUE = "2";
   private static final String TOTAL_RESPONSE_VALUE = "2";
   private static final String RESPONSE_CODE = "200";
-  private static final String TPFS_RESULT = "127.0.0.1:2";
+  private static final String TPFS_RESULT = "127.0.1.1:1\n127.0.0.1:2\n";
```

Comment:
Earlier, we were only reading one part file and verifying the result not all the part files.
Also, I have a question on this one:
Say, first line comes from part0 file which is written by reducer0 and second line comes from part1 which is written by reducer1. Can we assume that they will always be in this order ?